### PR TITLE
Add vit j

### LINKED ISSUE #11: Confirm vitamin-j is the correct group structure
... and add reference and/or appendix that contains specific structure

### PR DESCRIPTION
**Pull request by makeclean**
This fixes #11

### CODE REVIEW

File: src/report.tex [hunk 1]
```diff
@@ -706,7 +718,7 @@ \subsubsection{Neutron Transport}
 The two batches of calculations were performed, one
 case with the B$_4$C liner and one without.  
 In each case, the neutron flux was tallied in 175 energy groups using the
-vitamin-j structure.
+VITAMIN-J structure \cite{vitaminj}.
```
Comment (gonuke) [inline src/report.tex]: as shown in appendix \ref{....}

File: src/report.tex [hunk 2]
```diff
@@ -1889,7 +1901,57 @@ \subsection*{Variance Reduction}
 \clearpage
 \section{Appendix - A - Materials used in the problem}
 \input{materials.tex}
+\newpage
+\clearpage
+\section{Appendix - B - VITAMIN-J Group Boundaries}
```
Comment (gonuke) [inline src/report.tex:1904]: "[MeV]"

File: src/report.tex [hunk 3]
```diff
@@ -706,7 +718,7 @@ \subsubsection{Neutron Transport}
 The two batches of calculations were performed, one
 case with the B$_4$C liner and one without.  
 In each case, the neutron flux was tallied in 175 energy groups using the
-vitamin-j structure.
+VITAMIN-J structure \cite{vitaminj} as shown in Appendix B \ref{appendix_b}.
```
Comment (gonuke) [inline src/report.tex:722]: This will probably give the letter "B" twice.  just leave the \ref

**Comment by gonuke** [comment]
This may result in merge hell since it is based on #25 but that one has diverged now....

File: src/report.tex [hunk 4]
```diff
@@ -706,7 +718,7 @@ \subsubsection{Neutron Transport}
...
 
 %% NOTE [PPHW]: is vitamin=j correct here?
```
Comment (gonuke) [inline src/report.tex]: remove NOTE

**Comment by gonuke** [comment]
I'm going to rebase this....

**Comment by makeclean** [comment]
Ok this is good to go.

**Comment by gonuke** [comment]
Merging now, before it gets messier...

**Pull request merged**